src/styles/variables.css

```diff
@@ -1,5 +1,6 @@
 @custom-media --mobile (width <= 650px);
```

Comment:
pls add a comment noticing that changing of this value should be done with changing it at the `utils.ts@isMobile`